lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -14842,3 +14842,314 @@ msgstr "Eléments verrouillés"
+
+#:Centreon/Domain/ConfigurationLoader/YamlConfigurationLoader.php:102
+msgid "Loop detected in file %1"
+msgstr "Boucle détectée dans le fichier %1"
+
+#:Centreon/Domain/ConfigurationLoader/YamlConfigurationLoader.php:137
+msgid "The configuration file %1 does not exists"
+msgstr "Le fichier de configuration %1 n'éxiste pas"
+
+#:Centreon/Domain/Downtime/DowntimeService.php:233
+msgid "Downtime not found"
+msgstr "Plage de maintenance non trouvée"
+
+#:Centreon/Domain/Downtime/DowntimeService.php:239
+msgid "Downtime already cancelled for this %1"
+msgstr "Plage de maintenance déjà annulée pour %1"
+
+#:Centreon/Domain/Downtime/DowntimeService.php:267
+msgid "Incorrect Resource Type"
+msgstr "Type de ressource incorrect"
+
+#:Centreon/Domain/Engine/EngineService.php:76
+#:Centreon/Domain/Engine/EngineService.php:102
+#:Centreon/Domain/Engine/EngineService.php:169
+#:Centreon/Domain/Engine/EngineService.php:220
+msgid "The contact alias is empty"
+msgstr "Le \"contact alias\" est vide"
+
+#:Centreon/Domain/Engine/EngineService.php:79
+#:Centreon/Domain/Engine/EngineService.php:129
+#:Centreon/Domain/Engine/EngineService.php:175
+#:Centreon/Domain/Engine/EngineService.php:266
+#:Centreon/Domain/Engine/EngineService.php:316
+#:Centreon/Domain/HostConfiguration/HostConfigurationService.php:47
+msgid "Host name can not be empty"
+msgstr "Le nom d'hôte ne peut être vide"
+
+#:Centreon/Domain/Engine/EngineService.php:105
+#:Centreon/Domain/Engine/EngineService.php:149
+#:Centreon/Domain/Engine/EngineService.php:146
+msgid "The host of service is not defined"
+msgstr "L'hôte du service n'est pas défini"
+
+#:Centreon/Domain/Engine/EngineService.php:172
+msgid "Host of downtime not found"
+msgstr "L'hôte associé à la plage de maintenance non trouvé"
+
+#:Centreon/Domain/Engine/EngineService.php:235
+msgid "The host of service %1 is not defined"
+msgstr "L'hôte du service %1 n'est pas défini"
+
+#:Centreon/Domain/Engine/EngineService.php:238
+msgid "Host name of service %1 can not be empty"
+msgstr "Le nom d'hôte du service %1 ne peut pas être vide"
+
+#:Centreon/Domain/Engine/EngineService.php:241
+msgid "The description of service %1 can not be empty"
+msgstr "La description du service %1 ne peut pas être vide"
+
+#:Centreon/Domain/Engine/EngineService.php:286
+msgid "Host and service id can not be null at the same time"
+msgstr "L'ID de l'hôte et du service ne peuvent être null en même temps"
+
+#:Centreon/Domain/Engine/EngineService.php:289
+msgid "Downtime internal id can not be null"
+msgstr "L'ID de l'hôte et du service ne peuvent être null en même temps"
+
+#:Centreon/Domain/Engine/EngineService.php:355
+msgid "Host name cannot be empty"
+msgstr "Le nom d'hôte ne peut être vide"
+
+#:Centreon/Domain/Engine/EngineService.php:359
+msgid "Service description cannot be empty"
+msgstr "La description du service ne peut être vide"
+
+#:Centreon/Domain/Exception/ExpiredTokenException.php:29
+msgid "Token expired."
```

Comment:
Are . allowed in msgids?

Reply:
they actually are allowed, but it is not good practice to use them. We have several of this kind in the translation file and this is something that can be improved for sure.